tests-e2e/cypress/integration/channels/rhs/template_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+describe('channels > rhs > template', () => {
+    let team1;
+    let testUser;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            team1 = team;
+            testUser = user;
+        });
+    });
+
+    beforeEach(() => {
+        // # Size the viewport to show the RHS without covering posts.
+        cy.viewport('macbook-13');
+
+        // # Login as testUser
+        cy.apiLogin(testUser);
+    });
+
+    describe('create playbook', () => {
+        describe('open new playbook creation modal and navigates to playbooks', () => {
+            it('after clicking on Use', () => {
+                // # Switch to playbooks DM channel
+                cy.visit(`/${team1.name}/messages/@playbooks`);
+
+                // # Open playbooks RHS.
+                cy.get('button > .icon-product-playbooks').click();
```

Comment:
```suggestion
                cy.getPlaybooksAppBarIcon().should('be.visible').click();
```

since we're running CI with the app bar enabled now